SofascoreAcademy/SofascoreAcademy/Views/EventCellView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  EventCellView.swift
+//  SofascoreAcademy
+//
+//  Created by Matija Pavicic on 15.03.2024..
+//
+
+import Foundation
+import SnapKit
+import SofaAcademic
+import UIKit
+
+class EventCellView: BaseView {
+    
+    let timeCellView: TimeCellView = .init()
+    let homeTeamCellView: TeamCellView = .init()
+    let awayTeamCellView: TeamCellView = .init()
+    let homeScoreLabel: UILabel = .init()
+    let awayScoreLabel: UILabel = .init()
```

Comment:
Ovo ne moraš sad na ovoj zadaći, ali možemo početi primjenjivati na sljedećima...

Ovo bi sve trebale bitii private varijable, i onda da izvana samo njima pristupaš funkcijama za postavljanje vrijednosti koje ti trebaju... Ovako svako im može pristupiti za sve, i izavana može biti nejasno što im je potrebno mijenjati.